src/store/modules/target.js

```diff
@@ -507,6 +507,25 @@ export default ({
           })
       }
     },
+    removeSingleRootDomainFromServer ({ state, commit, rootState }, targetName) {
+      if (state.authentication_token !== '') {
+        const entity = rootState.general.entitiesToDelete[0]
+        return axios.delete('/rootdomains/deleteSubdomians/' + targetName + '/' + entity.name)
```

Comment:
this endpoint remove the list of subdomains, to delete de rootdomain we need to use
```DELETE api/rootdomains/{targetName}/{rootDomainName}```